### PR TITLE
Changes from AngularDocFX_EN

### PR DESCRIPTION
**Pull request by ESShared**

### CODE REVIEW

**Review by HUSSAR-mtrela — DISMISSED** [review]

**Comment by HUSSAR-mtrela** [comment]
@zdrawku please review and merged this PR

File: en/components/charts/chart-features.md [hunk 1]
```diff
@@ -253,4 +253,4 @@ Use trendlines to identify a trend or find patterns in your data. There are many
 
 ## API Members
 
--   [`IgxDataGridComponent`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igxdatagridcomponent.html)
```
Comment (zdrawku) [inline en/components/charts/chart-features.md:256]: @HUSSAR-mtrela this should lead to https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/IgxGridComponent.html

**Comment by zdrawku** [comment]
@HUSSAR-mtrela also there is a warning causing the build to fail:

image

**Comment by HUSSAR-mtrela** [comment]
@zdrawku I fixed above issue and I'm waiting for a new build [AngularDocFX_EN](http://tfs.infragistics.local:8080/tfs/Engineering/IgInternalApplicationsGit/IgInternalApplicationsGit%20Team/_build/index?context=allDefinitions&path=%5C&definitionId=1812&_a=completed) to update this PR

**Comment by zdrawku** [comment]
> @zdrawku I fixed above issues and I'm waiting for a new build [AngularDocFX_EN](http://tfs.infragistics.local:8080/tfs/Engineering/IgInternalApplicationsGit/IgInternalApplicationsGit%20Team/_build/index?context=allDefinitions&path=%5C&definitionId=1812&_a=completed) to update this PR

Okay

**Pull request merged**